backend/models/user.js

```diff
@@ -48,10 +49,26 @@ const FileRef = require("./fileRef");
 const PermissionTemplates = require("./permissionTemplates");
 const { get } = require("lodash");
 
+const appendRemainingLoginsInfo = function (resCode, remaining) {
```

Comment:
Instead of doing this? why don't we just have a function that spits out the object? And only call this function when you're certain you want to throw this error?

Currently it feels very hacky and trying to do too much. It should only call it when it needs to

```suggestion
return { 
    ...responseCodes.INCORRECT_USERNAME_OR_PASSWORD,
    message: `${responseCodes.INCORRECT_USERNAME_OR_PASSWORD.message}  (Remaining attempts: ${remaining})`
};
    
}
```